app/resources/view.html.template

```diff
@@ -290,55 +289,53 @@
             {% endif %}
 
           </div>
-
-          {% if env.enable_captcha and extend_url %}
-            {% if expiration_days %}
-              <div class="warning">
-                {% if expiration_days > 1 %}
-                  {% blocktrans %}
-                    Warning: this record will expire in less than {{expiration_days}} days.
-                  {% endblocktrans %}
-                {% else %}
-                  {% blocktrans %}
-                    Warning: this record will expire in less than a day.
+          <div class="links section">
```

Comment:
I don't think you need class="links" here? It seems each the link below is surrounded by <div class="links">